digits/dataset/images/classification/views.py

```diff
@@ -88,35 +88,70 @@ def from_folders(job, form):
+            'angle_max': form.augmentation_rotation_angle_max.data,
+            'probability': form.augmentation_rotation_probability.data
+        }
+
+    if form.has_augmentation_translation.data:
+        augmentation['translation'] = {
+            'dx_min': form.augmentation_translation_dx_min.data,
+            'dx_max': form.augmentation_translation_dx_max.data,
+            'dy_min': form.augmentation_translation_dy_min.data,
+            'dy_max': form.augmentation_translation_dy_max.data,
+            'probability': form.augmentation_translation_probability.data
+        }
+
     job.tasks.append(
             tasks.CreateDbTask(
-                job_dir     = job.dir(),
-                parents     = parse_train_task,
-                input_file  = utils.constants.TRAIN_FILE,
-                db_name     = utils.constants.TRAIN_DB,
-                backend     = backend,
-                image_dims  = job.image_dims,
-                resize_mode = job.resize_mode,
-                encoding    = encoding,
-                compression = compression,
-                mean_file   = utils.constants.MEAN_FILE_CAFFE,
-                labels_file = job.labels_file,
+                job_dir      = job.dir(),
+                parents      = parse_train_task,
+                input_file   = utils.constants.TRAIN_FILE,
+                db_name      = utils.constants.TRAIN_DB,
+                backend      = backend,
+                image_dims   = job.image_dims,
+                resize_mode  = job.resize_mode,
+                encoding     = encoding,
+                compression  = compression,
+                mean_file    = utils.constants.MEAN_FILE_CAFFE,
+                labels_file  = job.labels_file,
+                augmentation = augmentation,
                 )
             )
 
     if percent_val > 0 or form.has_val_folder.data:
         job.tasks.append(
                 tasks.CreateDbTask(
-                    job_dir     = job.dir(),
-                    parents     = val_parents,
-                    input_file  = utils.constants.VAL_FILE,
-                    db_name     = utils.constants.VAL_DB,
-                    backend     = backend,
-                    image_dims  = job.image_dims,
-                    resize_mode = job.resize_mode,
-                    encoding    = encoding,
-                    compression = compression,
-                    labels_file = job.labels_file,
+                    job_dir      = job.dir(),
+                    parents      = val_parents,
+                    input_file   = utils.constants.VAL_FILE,
+                    db_name      = utils.constants.VAL_DB,
+                    backend      = backend,
+                    image_dims   = job.image_dims,
+                    resize_mode  = job.resize_mode,
+                    encoding     = encoding,
+                    compression  = compression,
+                    labels_file  = job.labels_file,
+                    augmentation = {},
```

Comment:
You can just do here what you did below at line 171. I think it's better to keep the number of changed lines small than preserve indentation. I sort of regret hand-aligning indentation in general.